feature/platform/transceiver/zr_input_output_power_test/README.MD

```diff
@@ -0,0 +1,112 @@
+# gNMI-1.20: Telemetry: 400ZR RX input and TX output power telemetry values streaming. 
+
+## Summary
+
+Validate 400ZR optics modules report accurate RX input and TX output power
+telemetry values.
+
+As per CMIS ZR modules report two types of RX input power and one TX output
+power.
+* RX Signal Power
+  * Reports the actual signal power after filtering out any extra noise.
+  * Is mapped to /component/optical-channel/ full path shown below
+* RX Total Power
+  * Reports RX Signal Power plus noise without any filtering.
+  * Is mapped to /component/transceiver/physical-channel full path shown below
+* TX Output Power
+  * This is the total TX output power
```

Comment:
Is there such thing as signal power on the TX or just the total TX output power? I wasn't sure if CMIS or the hardware supported both.

Reply:
There is no Signal TX power defined in CMIS. This is primarily because on RX side we have a filter, power measure after the filter is Signal power and before the filter is total power. No such filter on TX side.

Reply:
Thanks for confirming.